sdks/go/sdk.go

```diff
@@ -42,6 +43,14 @@ type SDK struct {
 	alpha  *Alpha
 }
 
+// ErrorLog is a function to log the error.
+type ErrorLog func(error, string)
+
+// Logger is a pluggable function that outputs the error message to standard error.
+var Logger ErrorLog = func(err error, msg string) {
+	fmt.Fprintf(os.Stderr, "msg: %s\n", err)
```

Comment:
```suggestion
	fmt.Fprintf(os.Stderr, "%s: %s\n", msg, err)
```

Since the message isn't in the log.

Reply:
done!